spec/models/manageiq/providers/autosde/inventory_spec.rb

```diff
@@ -23,5 +23,11 @@
     expect(ems.cloud_volumes).to_not be_empty
 
     expect(ems.cloud_volumes.first).to be_instance_of(ManageIQ::Providers::Autosde::StorageManager::CloudVolume)
+
+    # wwpn candidates
+    expect(ems.wwpn_candidates).to_not be_empty
+    first_candidate = ems.wwpn_candidates.first
```

Comment:
:-1: for using `.first` unless you also explicitly expect the count to be `1` above, otherwise you're going to hit sporadic test failures.

Prefer to use `ems.wwpn_candidates.find_by(:ems_ref => "something")`

Reply:
Not sure why this was resolved, still an issue